package.json

```diff
@@ -100,11 +100,15 @@
     "sinon-chai": "3.7.0",
     "through2": "4.0.2",
     "touch": "3.1.0",
+    "tsify": "5.0.4",
+    "typescript": "4.9.4",
     "vinyl-buffer": "1.0.1",
     "vinyl-source-stream": "2.0.0",
     "watchify": "4.0.0"
   },
   "optionalDependencies": {
+    "@rollup/plugin-commonjs": "24.0.0",
```

Comment:
oh, why are these optional?

Reply:
The `check` and `unit` GitHub action checks can ignore these dependencies, to speed up their Install steps